wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -117,6 +119,24 @@ def _is_correct_base_class(self, base_class: ast.AST) -> bool:
             return len(subscripts) == 1 and correct_items
         return False
 
+    def _check_getters_setters_methods(self, node: ast.ClassDef) -> None:
+        class_attributes, instance_attributes = classes.get_attributes(node)
+        attribute_names = set(
+            name_nodes.flat_variable_names(class_attributes),
+        ).union({
+            instance.attr for instance in instance_attributes
+        })
+
+        for attribute in attribute_names:
+            for method_postfix in classes.getter_setter_postfixes(node):
+                if attribute.lstrip('_') == method_postfix:
```

Comment:
We can strip attribute names earlier. Because we can dedupe them in sets and not make extra loop iterations for cases like `_some`, `some`, and `__some`